src/components/TopBarMenu/Categories/Categories.jsx

```diff
@@ -0,0 +1,15 @@
+import "./Categories.scss";
+
+export default function Categories() {

```

Comment:
You're doing a navigation top bar, so you can move it to `./components`, and instead of calling it the `categories` the most common name it's `Navbar`. Categories is too specific and you can transform it into a reusable component. For example:

```jsx
const links = [
  { url: 'www.mysite.com/men', text: 'Men' },
  { url: 'www.mysite.com/women', text: 'Women' },
]

{/* You passed links via props and mount your links dynamic there */}
<Navbar links={links} />
```